cextern/wcslib/C/wcshdr.c

```diff
@@ -1942,45 +1943,50 @@ void wcshdo_format(
   char *format)
 
 {
-  char cval[24];
-  int  cp0, cpi, i, expmax, expon, nsig, precision;
-
-  if (fmt == 'G') {
-    fmt = 'f';
-    for (i = 0; i < nval; i++) {
-      if (fabs(val[i]) < 1e-4 || 1e12 < val[i]) {
-        fmt = 'E';
-        break;
-      }
-    }
-  }
-
-  cp0 = 2;
-  expmax = -999;
-  for (i = 0; i < nval; i++) {
-    /* Double precision has at least 15 significant digits, and up to 17:  */
-    /* http://en.wikipedia.org/wiki/Double-precision_floating-point_format */
+  int emax = -999;
+  int emin = +999;
+  int precision = 0;
+  for (int i = 0; i < nval; i++) {
```

Comment:
```suggestion
  int i;
  for (i = 0; i < nval; i++) {
```
This is to avoid compilation error.